lib/redis-session-store.rb

```diff
@@ -128,6 +128,11 @@ def set_session(env, sid, session_data, options = nil)
   end
   alias write_session set_session
 
+  def get_expiry(env, options)
+    opts = options || env.fetch(ENV_SESSION_OPTIONS_KEY)
```

Comment:
oh you are right, I was misreading what was up there before. My basic feeling on this is that having `options` and `opts` vars in the same method is tricky. If we can find slightly better names for one or both, that would be 🆒